src/ReverseProxy/Configuration/ConfigurationConfigProvider.cs

```diff
@@ -164,41 +165,44 @@ private Cluster CreateCluster(IConfigurationSection section)
 
         private static ProxyRoute CreateRoute(IConfigurationSection section)
         {
-            var route = new ProxyRoute
+            return new ProxyRoute
```

Comment:
Expression body maybe?

```suggestion
            => new ProxyRoute
```

Reply:
Only a few of these are concise enough for expression bodies and I'd rather use a consistent style.